include/net/download_client.h

```diff
@@ -81,10 +81,17 @@ struct download_client_evt {
  * @brief Download client configuration options.
  */
 struct download_client_cfg {
-	/** TLS security tag. If -1, TLS is disabled. */
+	/** IP port.
+	 *  Pass zero to use default, or non-zero to override.
+	 */
+	int port;
+	/** TLS security tag.
+	 * Pass -1 to disable TLS.
```

Comment:
nit: alignment

Reply:
The `*` alignment looks correct to me (`checkpatch` will complain if I align it with the second asterisk on the line above).

Reply:
I see now which alignment you meant :)